nautobot/core/models/generics.py

```diff
@@ -25,6 +31,38 @@ class Meta:
         abstract = True
 
 
+class _NautobotTaggableManager(_TaggableManager):
```

Comment:
Can you please add a docstring for this manager to explain why it is different from `TaggableManager` to describe the distinction between `TaggableManager` (field) and the need for overloading `_TaggableManager` (manager)?

Should we also have a `2.0 TODO` to rip this out in v2?

Reply:
Good call, and yes we should. I'll add it.